guides/release/testing/testing-tools.md

```diff
@@ -0,0 +1,185 @@
+Ember comes with great testing tools out of the box and popular add-ons which you can use as your testing game evolves.
```

Comment:
```suggestion
Ember comes with great testing tools out of the box and popular addons which you can use as your testing game evolves.
```

I know it seems a bit odd, but we use `addon` instead of `add-on` in our Guides. Once you fix this here and in the other places below, the linter test will pass.